diff -Naur gzip-1.5/zmore.in gzip-1.5-addsuffix/zmore.in
--- gzip-1.5/zmore.in	2012-01-01 08:53:58.000000000 +0000
+++ gzip-1.5-addsuffix/zmore.in	2012-12-06 13:57:04.339303272 +0000
@@ -70,7 +70,7 @@
         --v*) exec echo "$version";;
         esac
 
-        < "$FILE" || continue
+        if test -e $FILE || test -e $FILE.gz; then 
         if test $FIRST -eq 0; then
                 echo $n1 "--More--(Next file: $FILE)$n2"
                 stty $cb -echo 2>/dev/null
@@ -81,6 +81,8 @@
                 [eq]) exit;;
                 esac
         fi
+        else < $FILE                                                                                                       
+        fi 
         if test "$ANS" != 's'; then
                 echo "------> $FILE <------"
                 gzip -cdfq -- "$FILE" | eval ${PAGER-more}
